core/src/test/scala/org/apache/spark/executor/ProcfsMetricsGetterSuite.scala

```diff
@@ -38,4 +40,26 @@ class ProcfsMetricsGetterSuite extends SparkFunSuite {
     assert(r.jvmVmemTotal == 4769947648L)
     assert(r.jvmRSSTotal == 262610944)
   }
+
+  test("partial metrics shouldn't be returned") {
```

Comment:
Could you add a test prefix, `SPARK-34845: `?

Reply:
Done